contrib/initramfs/Makefile.am

```diff
@@ -1,4 +1,4 @@
-initrddir = $(datarootdir)/initramfs-tools
+initrddir = /usr/share/initramfs-tools
```

Comment:
Just one question, do we know of any distributions when this external package installs itself somewhere else?  Would it be worthwhile to provide a mechanism to set this path via configure?

Reply:
I am not aware of any distribution having anything else than `/usr/share`.

Debian does hard code it (see https://salsa.debian.org/kernel-team/initramfs-tools/blob/master/mkinitramfs), ubuntu does not change this. I could not find any debian based distribution with any other path